src/util.h

```diff
@@ -50,6 +50,11 @@ extern CTranslationInterface translationInterface;
 extern const char * const BITCOIN_CONF_FILENAME;
 extern const char * const BITCOIN_PID_FILENAME;
 
+extern fs::detail::utf8_codecvt_facet g_utf8;
+
+#define u8string() string(g_utf8)
+#define u8path(str) path(str, g_utf8)
```

Comment:
Is it better to do these as methods in the namespace? Preprocessor use should be minimized IMO.

Reply:
I just thought that pass the second parameter at every `.string()` and `path()` is pretty annoying and easy to forget. If most developers don't want to use macro function, I could do a scipted-diff.

Reply:
I mean define these macros as functions in the fs namespace that call through as defined here.

Reply:
We could include `u8path` into fs namespace, however for `u8string` we couldn't since it's a member function.

Reply:
I'll defer to others as to whether this is worthwhile, but absent other options, I would make a function ala `u8path` for string as well, and hide `g_utf8`